src/briefcase/integrations/android_sdk.py

```diff
@@ -1265,11 +1254,11 @@ def logcat(self):
                     "-s",
                     self.device,
                     "logcat",
-                    "-s",
-                    "MainActivity:*",
-                    "stdio:*",
-                    "Python:*",
-                ],
+                    "--pid",  # This option is available since API level 24.
+                    pid,
```

Comment:
I've updated https://github.com/beeware/briefcase-android-gradle-template/pull/52 to set the minimum API level to 24.